From e88e2ac317185b47a033e6b1b09782005756321a Mon Sep 17 00:00:00 2001
From: Alin Jerpelea <alin.jerpelea@sony.com>
Date: Sat, 26 May 2018 08:50:32 +0200
Subject: [PATCH] media: disabled functions that need blobs

The AOSP code should be compiled from code only vithout
vendor binaries

Test: manual - Build is ok without vendor blobs

Change-Id: I3a6ec444f732c5faf71a9d29a3116fc70bf07782
Signed-off-by: Alin Jerpelea <alin.jerpelea@sony.com>
---
 sdm845/mm-video-v4l2/vidc/venc/Android.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sdm845/mm-video-v4l2/vidc/venc/Android.mk b/sdm845/mm-video-v4l2/vidc/venc/Android.mk
index a4adec8..4af36dc 100755
--- a/sdm845/mm-video-v4l2/vidc/venc/Android.mk
+++ b/sdm845/mm-video-v4l2/vidc/venc/Android.mk
@@ -21,7 +21,7 @@ libmm-venc-def += -Werror
 libmm-venc-def += -D_ANDROID_ICS_
 
 TARGETS_THAT_USE_FLAG_MSM8226 := msm8226 msm8916 msm8909
-TARGETS_THAT_NEED_SW_VENC_MPEG4 := msm8909 msm8937 sdm845
+TARGETS_THAT_NEED_SW_VENC_MPEG4 := msm8909 msm8937
 TARGETS_THAT_NEED_SW_VENC_HEVC := msm8992
 TARGETS_THAT_SUPPORT_UBWC := msm8996 msm8998 sdm845
 TARGETS_THAT_SUPPORT_VQZIP := msm8996 msm8998
-- 
2.15.1

